libs/api/domains/ministry-of-justice/src/lib/ministryOfJustice.resolver.ts

```diff
@@ -31,42 +30,37 @@ export class MinistryOfJusticeResolver {
   @Query(() => AdvertResponse, {
     name: 'ministryOfJusticeAdvert',
   })
-  @Audit()
```

Comment:
Is it no longer desired to audit log these requests on our side?

Reply:
I need to be able to fetch this data in the web application, These audit lines were causing the requests to return 401 Unauthorized

Reply:
Ahh I see, then its all good.